packages/react-components/src/atoms/__tests__/Dropdown.test.tsx

```diff
@@ -151,3 +156,98 @@ it('gets greyed out when disabled', () => {
   ).not.toBe(silver.rgb);
   expect(getByRole('textbox')).not.toBeDisabled();
 });
+
+it('shows the field in red when required field not filled', async () => {
+  const { getByText, getByRole, rerender } = render(
+    <Dropdown
+      options={[{ value: 'LHR', label: 'Heathrow' }]}
+      required={true}
+      value=""
+    />,
+  );
+
+  userEvent.click(getByText('Select'));
+  userEvent.tab();
```

Comment:
Wasn't tab disabled? What does this do?

Reply:
the selection on tab is disabled. But pressing the tab still works.
On the test, the tab will unfocus the input and then asserts if the validation happened